src/frontend/components/SøknadsSteg/Utvidet-DinLivssituasjon/DinLivssituasjon.test.tsx

```diff
@@ -74,4 +76,12 @@ describe('DinLivssituasjon', () => {
         );
         expect(result).not.toBeNull();
     });
+    it('Viser spørsmål om er du separert, enke eller skilt om sivilstand GIFT', () => {
+        const { queryByText } = render(
+            <TestProvidereMedEkteTekster>
+                <DinLivssituasjon />
+            </TestProvidereMedEkteTekster>
+        );
+        expect(queryByText(/omdeg.separertellerskilt.spm/)).not.toBeInTheDocument();
```

Comment:
Ble litt forvirret her, det ser ut som testens beskrivelse er at spørsmålet skal vises, men så sjekker expecten at den ikke vises? 🤔

Reply:
Nice catch, byttet litt frem og tilbake på hva som skulle testes, så skal oppdatere tittelen 😊

Reply:
Kanskje det er lurt å teste begge varianter? at det vises når man er gift, og vises ikke når man ikke er gift?

Reply:
Hmm har nå prøvd å legge til en ny "søker" i testen med sivilstand GIFT for å teste dette, men klarer ikke få til at det blir to ulike sivilstander 🤔